models/articles.models.js

```diff
@@ -49,3 +49,15 @@ exports.selectCommentsByArticleId = (articleId) => {
         return rows
     })
 }
+
+exports.addCommentToArticle = (articleId, body) => {
+    return db.query(`
+        INSERT INTO comments
+        (body, author, article_id)
+        VALUES ($1, $2, $3)
+        RETURNING body`, [body.body, body.username, articleId])
```

Comment:
You can return the whole comment here, not just the body. If you were displaying this on the front end then you would want to display the username too and potentially use the article_id.